### PR TITLE
fix package path in go module and catch up to latest openapi spec

### PR DESCRIPTION
**Pull request by houqp**
This PR depends on https://github.com/apache/airflow/pull/9922.

### CODE REVIEW

**Comment by Fakerr** [comment]
Just curious here. Since this PR depends on another one, would you be interested by this [Github app](https://www.dpulls.com/) that I built that prevents PRs with "Depends on" from being merged before their dependencies ? Do you think it's something that can help you in your projects ? cc @houqp

**Comment by potiuk** [comment]
Linting problem :)

**Comment by houqp** [comment]
@potiuk yeah, not sure why the license headers inserted locally is not being accepted by the same command in CI. Digging into that right now.

**Comment by houqp** [comment]
@potiuk of course it's now working again after a rerun without any code change :P

**Comment by potiuk** [comment]
Obviously :)

File: airflow/README.md [hunk 1]
```diff
@@ -59,23 +59,23 @@ Class | Method | HTTP request | Description
...
 *DAGApi* | [**GetDagSource**](docs/DAGApi.md#getdagsource) | **Get** /dagSources/{file_token} | Get source code using file token
 *DAGApi* | [**GetDags**](docs/DAGApi.md#getdags) | **Get** /dags | Get all DAGs
-*DAGApi* | [**GetTask**](docs/DAGApi.md#gettask) | **Get** /dags/{dag_id}/tasks/{task_id} | Get simplified representation of a task.
+*DAGApi* | [**GetTask**](docs/DAGApi.md#gettask) | **Get** /dags/{dag_id}/tasks/{task_id} | Get simplified representation of a task
 *DAGApi* | [**GetTasks**](docs/DAGApi.md#gettasks) | **Get** /dags/{dag_id}/tasks | Get tasks for DAG
 *DAGApi* | [**PatchDag**](docs/DAGApi.md#patchdag) | **Patch** /dags/{dag_id} | Update a DAG
 *DAGApi* | [**PostClearTaskInstances**](docs/DAGApi.md#postcleartaskinstances) | **Post** /dags/{dag_id}/clearTaskInstances | Clears a set of task instances associated with the DAG for a specified date range.
+*DAGRunApi* | [**AirflowApiConnexionEndpointsDagRunEndpointPostDagRun**](docs/DAGRunApi.md#airflowapiconnexionendpointsdagrunendpointpostdagrun) | **Post** /dags/{dag_id}/dagRuns | Trigger a DAG Run
```
Comment (mik-laj) [inline airflow/README.md:69]: Is it expected? Shouldn't we fix it in the specification?

Comment (houqp) [reply airflow/README.md:69]: Yeah, this needs to be fixed in the api spec.

**Pull request merged**